ci-operator/infra/openshift/release-controller/releases/release-ocp-4.3-amd64.json

```diff
@@ -0,0 +1,82 @@
+{
```

Comment:
We already have this, it’s the current default.  Why add another?

Reply:
For multiarch it makes automation simpler. No special `default` case. Just a loop over your known arches.

Reply:
That is to say, we treat all supported arches equally.

Reply:
other thing: though it's not on the radar now, we have had discussions about having a manifest-list enabled stream in the future. It figured that would be the stream that doesn't include the architecture suffixes.